SPECS/compiler-rt/compiler-rt.spec

```diff
@@ -0,0 +1,72 @@
+%global maj_ver 12
```

Comment:
Do we need all the Macros?

Reply:
This spec is designed to mimic the clang spec (which is its cousin, given it comes from the same llvm repo) - this is how the clang spec specified its version. If you want me to simplify it that makes sense but shouldn't these be similar to each other?

Reply:
Since these 3 macros are used only once, there's really no gain here, so removing them will reduce clutter.

When it comes to similarity: keeping specs similar to the original we used from a different distro may make future updates easier. That mostly applies to the build, installation, test, and file sections, though. Here we can just drop these macros.